parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -138,6 +139,11 @@ public WriteBuilder metricsConfig(MetricsConfig newMetricsConfig) {
       return this;
     }
 
+    public WriteBuilder writeMode(ParquetFileWriter.Mode newWriteMode) {
```

Comment:
Instead of passing in a write mode, I think Parquet should use the same methods as the other file format helpers, `overwrite()` and `overwrite(boolean enabled)`. That way, all of them have a similar API and we don't leak Parquet classes through the Iceberg API.

Reply:
Agree, made the changes.